common/Perf/Azure.Test.Perf/PerfOptions.cs

```diff
@@ -25,6 +25,18 @@ public class PerfOptions
         [Option('l', "latency", HelpText = "Track and print per-operation latency statistics")]
         public bool Latency { get; set; }
 
+        [Option("max-completion-port-threads", HelpText = "The maximum number of asynchronous I/O threads that the thread pool creates on demand")]
+        public int? MaxCompletionPortThreads { get; set; }
```

Comment:
I understand the formal parameter names for the `ThreadPool` setter calls are `completionPortThreads`, but could we name these Min/Max`IOCompletionThreads`.
The [C# sample page](https://docs.microsoft.com/en-us/dotnet/api/system.threading.threadpool.setminthreads?view=net-5.0) for the API also agrees.

Reply:
Sure, I have no strong feelings about the names.